platform/commonUI/browse/res/templates/browse.html

```diff
@@ -24,7 +24,7 @@
     <mct-include key="'topbar-browse'"></mct-include>
     <div class="abs holder holder-main browse-area s-browse-area browse-wrapper"
          ng-controller="PaneController as modelPaneTree"
-         ng-class="modelPaneTree.visible() ? 'pane-tree-showing' : 'pane-tree-hidden'">
+         ng-class="modelPaneTree.visible() ? 'pane-tree-showing' : 'pane-tree-hidden'" alias="Tree">
```

Comment:
Hrm, `alias="Tree"` doesn't clearly communicate what this does. 

How about something like `hide-parameter="hideTree"`